src/pages/Patch.tsx

```diff
@@ -1,11 +1,27 @@
 import React from "react";
 import { BreadCrumb } from "components/Breadcrumb";
+import { BigTitle } from "components/styles/Typography";
+import { PageWrapper } from "components/styles/PageWrapper";
+import { SiderCard } from "components/styles/SiderCard";
+import { PageHeader } from "components/styles/PageHeader";
+import { PageContent, PageLayout, PageSider } from "components/styles/Layout";
 
 export const Patch = () => {
   return (
-    <>
+    <PageWrapper>
       <BreadCrumb displayName="Specific Patch" />
-      <div>Patch Page</div>
-    </>
+      <PageHeader>
+        <BigTitle>Current Patch Name</BigTitle>
```

Comment:
I would pull this into the metadata card. I don't think the patch name needs to be prominent on the page and this will leave more space for the main pane

Reply:
going with designs! talk to ben for changes :)